src/test/kotlin/org/jetbrains/research/transformations/SetupParser.kt

```diff
@@ -0,0 +1,50 @@
+package org.jetbrains.research.transformations
+
+import java.io.File
+import java.util.logging.Logger
+
+object SetupParser {
```

Comment:
Please, leave a comment describing why and when do we need to setup parser. You can add dokka library to document your code properly, please, see https://kotlinlang.org/docs/reference/kotlin-doc.html. You can omit to describe all parameters or return types, but sometimes it's nice to refer some classes or fields in comments.